docs/source/deployorderer/ordererchecklist.md

```diff
@@ -308,6 +310,57 @@ Because Prometheus utilizes a "pull" model there is not any configuration requir
 * **`Provider`**: Set this value to `statsd` if using `StatsD` or `prometheus` if using `Prometheus`.
 * **`Statsd.Address`**: (required to use `StatsD` metrics for the ordering node) When `StatsD` is enabled, you will need to configure the `hostname` and `port` of the `StatsD` server so that the ordering node can push metric updates.
 
+## Admin.*
+
+```
+Admin:
+    # host and port for the admin server
+    ListenAddress: 127.0.0.1:9443
+
+    # TLS configuration for the admin endpoint
+    TLS:
+        # TLS enabled
+        Enabled: false
+
+        # Certificate is the location of the PEM encoded TLS certificate
+        Certificate:
+
+        # PrivateKey points to the location of the PEM-encoded key
+        PrivateKey:
+
+        # Most admin service endpoints require client authentication when TLS
+        # is enabled. ClientAuthRequired requires client certificate authentication
+        # at the TLS layer to access all resources.
+        #
+        # NOTE: When TLS is enabled, the admin endpoint requires mutual TLS. The
+        # orderer will panic on startup if this value is set to false.
+        ClientAuthRequired: true
+
+        # Paths to PEM encoded ca certificates to trust for client authentication
+        ClientRootCAs: []
+```
+
+* **`ListenAddress`**: The orderer admin server address (host and port) that can be used by the `osnadmin` command to configure channels on the ordering service. This value should be a unique `host:port` combination to avoid conflicts.
+* **`TLS.Enabled`**: Technically this can be set to `false`, but this is not recommended. In general, you should always set this value to `true`.
+* **`TLS.Certificate`**: The path to and file name of the orderer signed certificate issued by the TLS CA.
+* **`TLS.PrivateKey`**: The path to and file name of the orderer private key issued by the TLS CA.
+* **`TLS.ClientAuthRequired`**: This value must be set to `true`. Note that while mutual TLS is required for all operations on the orderer `Admin` endpoint, the entire network is not required to use mutual TLS.
+* **`TLS.ClientRootCAs`**: The path to and file name of the admin client TLS CA root certificate.
+
+## ChannelParticipation.*
+
+```
+ChannelParticipation:
+    # Channel participation API is enabled.
+    Enabled: false
+
+    # The maximum size of the request body when joining a channel.
+    MaxRequestBodySize: 1 MB
+```
+
+* **`Enabled`**: If you are bootstrapping the ordering node with a system channel genesis block, this should remain `false`. If you are **not** bootstrapping the ordering node with a system channel genesis block, this should be overridden to `true`. Make sure that your [`General.BoostrapMethod`](#general-boostrapmethod) is set to `none` if `ChannelParticipation.Enabled` is set to `true`, otherwise you will get an error when you attempt to start the node.
```

Comment:
Speaking of... where did this come from? I don't think anything's changed that would result in an error at startup with the channel participation API enabled and the file bootstrap method.

Reply:
I can't recall where this came from, exactly. I thought I heard it somewhere, but I suppose it's possible I just assumed it. 

I assume this general guidance here is correct, and that this is at least recommended. I'll edit this to remove everything starting with "otherwise". Let me know if I should go farther.

Reply:
I don't see any reason to avoid enabling channel participation even if they're still using a system channel (either they take advantage of the new list channel features or they don't), but there's nothing wrong with the advice to not enable it. Any thoughts here @jyellick or @denyeart?

Reply:
I suppose I'd encourage users to enable the channel participation API, if for no other reason than giving them the ability to list channels and migrate away.  I wouldn't say it's critical, and it's fine not to enable it.  I suppose we're looking to give concrete advice on "this should be enabled, or this should not be enabled" though, so I would err towards turn it on.

Reply:
I'd suggest remove all the extra language and just say to enable it if you'd like to use the channel management APIs.

Reply:
The reason we disabled it in the defaults, is because Admin TLS is also disabled by default. We didn't want an unsecured service. So I guess we should at least say if you enable it, make sure you've enabled Admin TLS.